README.md

```diff
@@ -61,6 +61,21 @@ To make writing this kind of `nav` more natural ([in YAML there's no better opti
     * [Foo](borgs/foo.md)
 ```
 
+### Specifying the page to use for the section
+
+By default the first child is used as the section page, even if there is no
+`index.md`. If you want to change the page to use add the `use-page` config
+option:
+
+```yaml
+plugins:
+  - section-index:
+    use-page: index
```

Comment:
* Underscores are much more typical for options.
* For some reason I think it's really about files, not pages.
* Using just the stem of the file name is confusing.
* Needs an extra indent.

With all that together, this is what I would do:

```suggestion
      index_page: index.md
```

Reply:
Oh sorry, I wrote this wrong.
I would really prefer to name it <code>index_<b>file</b></code>
```suggestion
      index_file: index.md
```